README.md

```diff
@@ -163,6 +163,7 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
 
 
    **Note** - The defaults of the install script can be overridden see the build in help.
```

Comment:
It looks like we still have this lingering line 😄 
```suggestion
```

Reply:
Isn't that already fixed 👀